src/L6/datasorter.py

```diff
@@ -91,7 +92,7 @@ def execute_merge_sort(self):
         #  and to store the value of datetime.now() in self.start_time and
         #  self.end_time at appropriate times
 
-        # First validate that the data is valid, if no ValueError is raised, we'll continue
+        # First validate that the data is valid, if no InvalidInputListError is raised, we'll continue
```

Comment:
"if not" sounds more natural.

Reply:
You mean in the comment? Grammatically speaking, "no error is raised" is more correct than "not error is raised", please refer to https://www.linguee.com/english-spanish/translation/if+no+error.html which compiles examples of this sentence in various sources online.